package.json

```diff
@@ -42,19 +42,16 @@
     "promise": "^8.0.1"
   },
   "devDependencies": {
+    "@opentok/client": "^2.18.0",
```

Comment:
@LukaszPilarski it should be the latest 2.31 https://github.com/opentok/opentok-network-test-js/blob/0d47bb539d9b12be63241436a9e4bc6edf39a34a/package.json#L45